modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,26 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premium upgrade promotion defaults
+
+When {productname} 6.2 or later is running in self-hosted Community mode, an *Upgrade* promotion button appears in the top-right of the editor menu bar by default. The button does note appear when {productname} is running in Community mode on the Tiny cloud.
+
+This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of the variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.
+
+When {productname} 6.2 or later is running in Premium mode, the *Upgrade* promotion button is disabled. And it is disabled when running on a Premium plan whether {productname} is running in the {cloudname} or as a self-hosted instance.
+
+To turn this button off in Community mode, set the `promotion` option, which defaults to `true`, to false.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  promotion: false
+});
+----
+
+The *Upgrade* button appears as part of the {productname} menu bar and, consequently, does not appear if the menu bar is disabled.
```

Comment:
Above, we said: 

> appears in the top-right of the editor menu bar

Should we keep the same phrasing?

Reply:
Works for me: done.

Reply:
Will it matter that it's in not in the top-right for RTL modes? In that case it's in the top-left.

Reply:
> Will it matter that it's in not in the top-right for RTL modes? In that case it's in the top-left.

Good point.

How does _appears in the far corner of the editor menu bar_ read to you?

Reply:
I’ve gone, at least in this moment, for _unused corner_.

Reply:
I have no strong opinions here, so whichever you think is best 😄